eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -86,14 +87,15 @@ services:
             - { name: liip_imagine.cache.resolver, resolver: ezpublish }
 
     ezpublish.image_alias.imagine.cache_resolver_decorator_factory:
-        class: '%ezpublish.image_alias.imagine.decorated_cache_resolver_factory.class%'
+        class: '%ezpublish.image_alias.imagine.cache_resolver_decorator_factory.class%'
         arguments:
             - '@ezpublish.config.resolver'
             - '@ezpublish.image_alias.imagine.cache_resolver_decorator.inner'
-            - '%liip_imagine.cache_resolver.proxy.class%'
+            - '%ezpublish.image_alias.imagine.cache_resolver_decorator_proxy.class%'
+            - '%ezpublish.image_alias.imagine.cache_resolver_decorator_relative.class%'
 
     ezpublish.image_alias.imagine.cache_resolver_decorator:
-        class: '%liip_imagine.cache_resolver.proxy.class%'
+        class: '%ezpublish.image_alias.imagine.cache_resolver_decorator_proxy.class%'
```

Comment:
should this rather be marked as abstract , and/or point to  ResolverInterface given it depends on factory?

Or the new relative resolver should extend the one from liip?

Reply:
It can't be marked as `abstract` because of:
```
  [Symfony\Component\DependencyInjection\Exception\RuntimeException]
  The definition "liip_imagine.cache.manager" has a reference to an abstract definition "ezpublish.image_alias.imagine.cache_resolver_decorat
  or". Abstract definitions cannot be the target of references.
```

but pointing it to an interface does the job, changed in: https://github.com/ezsystems/ezpublish-kernel/pull/2511/commits/aa9ad7e8b618e98da3f9e4f8fb260f62c612457b

Anyway, I think we should remove it while merging it up.